### PR TITLE
Add checks catalog view

### PR DESCRIPTION
**Pull request by arbulu89**
Checks catalog view.

I have changed the catalog json struct. Have a look please. If you like it, I would change the runner to create that struct.
Now, the checks catalog can be updated based on the provider. The data is loaded at initialization, so we wouldn't have real updates if the catalog is changed in the runner.

**I didn't update the package-lock.json file, as I think I have a different npm version, and it changes the whole file...***
**This is just a frontend work. It doesn't do anything in the backend. We will discuss about this afterwards**.
catalog

### CODE REVIEW

**Comment by stefanotorresi** [comment]
@arbulu89 would it be possible to preserve the code blocks around the parameters like it was before?

**Comment by arbulu89** [comment]
> @arbulu89 would it be possible to preserve the code blocks around the parameters in the description, like it was before?

Do you mean in the main description text? Yes, of course

File: assets/js/components/ChecksCatalog/ChecksCatalog.jsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  useEffect(() => {
+    setSelected(providers[0]);
+  }, [providers[0]]);
+
+  return (
+    <div>
+      <ProviderSelection
+        providers={providers}
+        selected={selected}
+        setSelected={setSelected}
```
Comment (dottorblaster) [inline assets/js/components/ChecksCatalog/ChecksCatalog.jsx:25]: call this prop `onChange` :wink:

**Review by dottorblaster — APPROVED** [review]
LGTM, if you want to address the tiny comment I made... otherwise it's alright :D

File: assets/js/components/ChecksCatalog/ChecksCatalog.jsx (hunk 1 above)
Comment (arbulu89) [reply assets/js/components/ChecksCatalog/ChecksCatalog.jsx]: Thanks!
Done: https://github.com/trento-project/web/pull/241/commits/2204511c4bde44584b72ad4dcd3afbf4d7fcda1d

**Review by nelsonkopliku — APPROVED** [review]

**Pull request merged**